recipes/tinyalsa/all/conanfile.py

```diff
@@ -17,17 +19,22 @@ class TinyAlsaConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
+    def export_sources(self):
```

Comment:
Prefer `exports_sources = ["patches/*"]`

Reply:
Currently, patch files are only available for 2.0.0.
Since this patch for 2.0.0 has already been merged into the main branch of tinyalsa, this patch is not needed for the feature release of tinyalsa.
So I am going to copy the patch file only for 2.0.0.

If that is inconvenient for you, please let me know why and I will try to fix it.
I will try to match the review as much as possible.

Reply:
It is an expected use-case in Conan. Patches are specified per-version - [example](https://github.com/conan-io/conan-center-index/blob/master/recipes/wt/all/conandata.yml) - so you can specify that 2.0.0 requires this patch and 2.0.1 does not.

Reply:
OK.I agree with you.
I renamed  "patches/tinyalsa-2.0.0-strncpy-warning.patch".
How about it?

Reply:
Great! Now you can export all patches via `export_sources = ["patches/*"]` without manual copying.

Reply:
Fixed it!